include/opentelemetry.hrl

```diff
@@ -50,7 +50,9 @@
           %% this field is not propagated and is only here as an implementation optimization
           %% If true updates like adding events are done on the span. The same as if the
           %% trace flags lowest bit is 1 but simply not propagated.
-          is_recorded       :: boolean() | undefined
+          is_recorded       :: boolean() | undefined,
+          %% parent span
```

Comment:
Nice, i was going to replace the 2-tuple with a new record `trace_ctx` that held the current span and the parent `trace_ctx` but this is better. It just needs a more detailed comment here to make clear why this is in the span context.